api/v2.0/swagger.yaml

```diff
@@ -991,6 +991,12 @@ paths:
           type: boolean
           required: false
           default: false
+        - name: with_sbom_overview
+          in: query
+          description: Specify whether the generate sbom overview is include inside the returning artifacts
```

Comment:
Specify whether the SBOM overview is included with the returning artifacts.

Reply:
It is not clear from the description what this "**overview**" it? Can you elaborate on that? As a consumer of the API, I would assume there is also a more detailed response available.